pandas/core/series.py

```diff
@@ -1132,6 +1132,37 @@ def count(self, level=None):
 
         return notnull(_values_from_object(self)).sum()
 
+    def value_counts(self, normalize=False, sort=True, ascending=False,
+                     bins=None, base=None):
+        """
+        Returns Series containing counts of unique values. The resulting Series
+        will be in descending order so that the first element is the most
+        frequently-occurring element. Excludes NA values
+
+        Parameters
+        ----------
+        normalize : boolean, default False
+            If True then the Series returned will contain the relative
+            frequencies of the unique values.
+        sort : boolean, default True
+            Sort by values
+        ascending : boolean, default False
+            Sort in ascending order
+        bins : integer, optional
+            Rather than count values, group them into half-open bins,
+            a convenience for pd.cut, only works with numeric data
+        base : list-like, optional
+            Unique values to count against
```

Comment:
can you document what happens if not every unique value is found in base?

Also, perhaps a name like `uniques` would be more evocative? The name `base` is not very intuitive to me.